docs/telemetry.md

```diff
@@ -1,12 +1,111 @@
+You can choose not to share this information. Participation in this program is completely voluntary. If you're uncomfortable sharing this anonymous data, you can [disable telemetry].
 
-At this time, telemetry is added only to the Percona packages and Docker images. Percona Server for MySQL collects only information about the installation environment. Future releases may add additional metrics.
+Your privacy is protected. Percona doesn't collect any personal information about you. All the data they collect is anonymous, meaning it can't be traced back to any individual user.
 
-Be assured that access to this raw data is rigorously controlled. Percona does not collect personal data. All data is anonymous and cannot be traced to a specific user. To learn more about our privacy practices, read our [Percona Privacy statement].
+We have a Percona Privacy statement explaining how we handle your data in more detail.
+
+Right now, this telemetry feature is only included in software packages and Docker images.
+
+## Telemetry components
+
+Percona collects information using the following components:
+
+* The Percona Telemetry DB part collects the necessary metrics directly from the database and stores them in a Metrics file.
+
+* The Metrics file stores the metrics and is a standalone file located on the database host's file system.
+
+* The Telemetry Agent is an independent process running on your database host's operating system and carries out the following tasks:
+
+    * Collects OS-level metrics
+
+    * Reads the Metrics file, adds the OS-level metrics
+
+    * Sends the full set of metrics to the Percona Platform
+
+The telemetry also uses the Percona Platform with the following components:
+
+* Telemetry Service - offers an API endpoint for sending telemetry. The service handles incoming requests. This service saves the data into Telemetry Storage.
+
+* Telemetry Storage - stores all telemetry data for the long term.
+
+## DB component
+
+The DB component extends the functionality of the database. This functionality can be implemented in the following ways, depending on the database:
+
+* Modifying the source code
+
+* Add modular components
+
+* Add self-contained extensions
+
+In Percona Server for MySQL, we have a DB component. The DB component is enabled by default during the initial database deployment. 
+
+This component does not collect the following information:
+
+* database names
+
+* user names or credentials
+
+* any user-entered value
+
+The component sends metrics gathered from the database instance to the Metrics file. This information is collected, by default, every 24 hours. Each connection creates a new Metrics file. When the file date exceeds seven days, the component removes the outdated files before creating the new file.
+
+The component creates a file in the local file system using a timestamp as the name with a `.tmp` extension, transfers the metrics to this file, and then renames the extension to .json.
```

Comment:
No need to mention about tmp files. It is implementation detail. Mention just about json files.